homeassistant/components/keenetic_ndms2/manifest.json

```diff
@@ -3,7 +3,7 @@
   "name": "Keenetic NDMS2 Router",
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/keenetic_ndms2",
-  "requirements": ["ndms2_client==0.1.1"],
+  "requirements": ["ndms2_client==0.1.2"],
```

Comment:
I'm missing a tag on github for this release:

https://github.com/foxel/python_ndms2_client/tags

Please tag the commit corresponding to the release published to PyPI to make it easier for the community to track changes to the library. Thanks!

Reply:
Sorry. Missed pushing the tag. https://github.com/foxel/python_ndms2_client/releases/tag/0.1.2